polaris-icons/icons/StatusActiveMajor.yml

```
@@ -0,0 +1,12 @@
+name: active status
+set: major
+description: Used to represent a resource that is in an Active state. 
+keywords:
+  - active
+  - tick
+authors:
+  - Brandon Law
+version: 1
+date_added: 2022-06-30
+date_modified: 2022-06-30
```

```suggestion
date_added: 2022-07-14
date_modified: 2022-07-14
```
Should this be updated to reflect closer to the shipping date? Or was this the date it was added to figma? I'm fine if you don't change this it's not a blocker